pytest_tests/testsuites/services/http_gate/test_http_system_header.py

```diff
@@ -368,9 +390,17 @@ def test_http_rfc_object_unavailable_after_expir(
             assert (
                 get_epoch(self.shell, self.cluster) == expected_epoch + 1
             ), f"Epochs should be equal: {get_epoch(self.shell, self.cluster)} != {expected_epoch + 1}"
-            try_to_get_object_and_expect_error(
-                cid=user_container,
-                oid=oid,
-                error_pattern="404 Not Found",
-                endpoint=self.cluster.default_http_gate_endpoint,
-            )
+
+            with allure.step("Check object deleted because it expires-on epoch"):
+                align_epochs(self.shell, self.cluster)
```

Comment:
if we add this to tick_epoch instead we don't need to check it every time in each test.